specification/data-resource-semantic-conventions.md

```diff
@@ -70,6 +70,28 @@ Labels defining a compute unit (e.g. Container, Process, Lambda Function).
 | container.image.name | Name of the image the container was built on. | `gcr.io/opentelemetry/operator` |
 | container.image.tag | Container image tag. | `0.1` |
 
+
+### Function as a Service
+
+**type:** `faas`
+
+**Description:** A serverless instance.
+
+| Label  | Description  | Example  | Required |
+|---|---|---|--|
+| faas.region| A specific geographical location where the function is executed. | `us-east-2` | Yes |
+| faas.name | The name of the function being executed. | `opentelemetry-lambda` | Yes |
+| faas.identifier | The unique name of the function being executed. <br /> For example, in AWS this field correspond to the [ARN] value, in GCP to the URI of the resource, and Azure to the [FunctionDirectory] field. | `opentelemetry-lambda` | Yes |
+| faas.version | The version of the function being executed. | `opentelemetry-latest` | No |
```

Comment:
Versions in the spec typically follow the format `semver:0.1.2`, `git:a4c2ef`, etc.

Reply:
The description leaves it ambiguous to what is meant by version. Is this the version of the lambda service, the version of the end user's application, or the version of the telemetry library?

Reply:
I agree, we should emphasize the semantic versioning. Furthermore, this comment made me thinking... Should we also cover information about the runtime in which the function is executed? For instance OS, language runtime version, etc..

Reply:
I think the latter (OS, ...) are part of resources and the semantic conventions around those.